src/linear_algebra/basic.lean

```diff
@@ -1835,6 +1835,15 @@ def of_submodules (p : submodule R M) (q : submodule R M₂) (h : p.map ↑e = q
 @[simp] lemma of_submodules_symm_apply {p : submodule R M} {q : submodule R M₂}
   (h : p.map ↑e = q) (x : q) : ↑((e.of_submodules p q h).symm x) = e.symm x := rfl
 
+/-- Given `f : M₁ ≃ₗ[R] M₂` and `U` a submodule of `M₂`, `f.comap U` is the
+induced `linear_equiv` from `U.comap f.to_linear_map` to `U`. -/
+@[simps]
```

Comment:
Are you sure `simps` generates anything useful here?

Reply:
iirc it used to generate an apply lemma and a sym_apply lemma but it seems they are no longer generated. Perhaps its because of the semiring?

Reply:
ok, I've added them manually now